src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/OrderPreferences/GeneralType.php

```diff
@@ -99,6 +99,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'label' => $this->trans('Recalculate shipping costs after editing the order', 'Admin.Shopparameters.Feature'),
                 'help' => $this->trans('Automatically updates the shipping costs when you edit an order.', 'Admin.Shopparameters.Help'),
+            ])
+            ->add('autoremove_not_valid_cartrules', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Autoremove cart rules not valid from cart', 'Admin.Shopparameters.Feature'),
```

Comment:
```suggestion
                'label' => $this->trans('Automatically remove invalid promo codes from the cart', 'Admin.Shopparameters.Feature'),
```

Hi, here's my wording suggestion. I think we should better use "promo code" to keep consistency with the vocabulary used in the front office. 
![image](https://user-images.githubusercontent.com/70583503/146033720-8add1929-f190-4f0b-8be0-c973781e532d.png)